pipelines/e2e/e2e.yaml

```diff
@@ -36,6 +36,10 @@ spec:
       type: string
       description: Public key to use for validation by the enterprise contract
       default: k8s://openshift-pipelines/public-key
+    - name: enterpriseContractExtraRuleData
```

Comment:
This new parameter will have to be provided by the release operator when creating the pipeline run. Do you intend to also modify https://github.com/konflux-ci/release-service to support this?

Reply:
@mmalina What exactly would need to be changed? I was poking around that repo and found a few specific parameters set in https://github.com/konflux-ci/release-service/blob/main/tekton/utils/pipeline_run_builder.go, but it seems like the rest are done dynamically. Am I missing something here?

Reply:
Yes, we pass few variables to the pipelines (most of them we get from the custom resources so we just pass the CR namespaced names then read from them in the tasks). If this is a pipeline parameter, the operator has to pass it when creating the pipelinerun

Reply:
Yeah, what Johnny says.